x-pack/test/api_integration/apis/endpoint/alerts.ts

```diff
@@ -27,7 +27,7 @@ export default function({ getService }: FtrProviderContext) {
   const nextPrevPrefixPageSize = 'page_size=10';
   const nextPrevPrefix = `${nextPrevPrefixQuery}&${nextPrevPrefixDateRange}&${nextPrevPrefixSort}&${nextPrevPrefixOrder}&${nextPrevPrefixPageSize}`;
 
-  describe('Endpoint alert API', () => {
+  describe.only('Endpoint alert API', () => {
```

Comment:
Don't forget to remove the `only`

Reply:
yah, i thought it would have been caught in linting before the commit was successful, how did that even happen?

Reply:
pushed it up righter, now.  i think i mis-understood the git processing I did... it didn't push the latest change after the commit passed, I see the mistake now tho I don't truly 'get' git in this regard, lol.  thanks P.